src/test/java/org/openhab/tools/analysis/checkstyle/test/BundleVendorCheckTest.java

```diff
@@ -0,0 +1,94 @@
+package org.openhab.tools.analysis.checkstyle.test;
+
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+import com.puppycrawl.tools.checkstyle.TreeWalker;
+import com.puppycrawl.tools.checkstyle.api.Configuration;
+import org.junit.Before;
+import org.junit.Test;
+import org.openhab.tools.analysis.checkstyle.BundleVendorCheck;
+import org.openhab.tools.analysis.checkstyle.api.AbstractStaticCheckTest;
+
+/**
+ * Tests for {@link BundleVendorCheck}
+ *
+ * @author Martin van Wingerden
+ */
+public class BundleVendorCheckTest extends AbstractStaticCheckTest {
+    private static final String TEST_DIRECTORY = "correctOpenHABSpellingInManifestTest/";
+    private DefaultConfiguration checkConfig;
+
+    @Before
```

Comment:
Don't you use the same configuration for every test? If that is the case can you please make the annotation of the method `@BeforeClass`, so that we create this configuration just one time?